src/components/pages/import-batches.yml

```diff
@@ -37,3 +37,6 @@ createImport:
 importsHeaderPrefix:
   en: Source
   de: Quelle
+noImportSourcesConfigures:
```

Comment:
Just a small typo: "Configures" instead of "Configured". I'll approve, so you can fix and merge immediately